tests/test_matching.py

```diff
@@ -177,37 +189,68 @@ def test_setup_matching(self, client, test_client, helpers):
             label = 'Low'
         ))
 
-        ## Set up matching
-        conference.setup_matching()
+        # Set up reviewer matching
+        conference.setup_matching(
+            conference.client.get_group(conference.get_reviewers_id()))
+
+        reviewer_matching_ids = _matching_invitation_ids(
+            client, conference.get_reviewers_id())
+
+        assert client.get_invitation(id=reviewer_matching_ids['assignment_config'])
+        assert client.get_invitation(id=reviewer_matching_ids['custom_load'])
+        assert client.get_invitation(id=reviewer_matching_ids['conflict'])
+        assert client.get_invitation(id=reviewer_matching_ids['aggregate_score'])
+        assert client.get_invitation(id=reviewer_matching_ids['paper_assignment'])
+
+        # Set up AC matching
+        conference.setup_matching(
+            conference.client.get_group(conference.get_area_chairs_id()))
 
-        assert client.get_invitation(id = 'auai.org/UAI/2019/Conference/-/Reviewing/Assignment_Configuration')
```

Comment:
I would prefer to be more explicit when I do an assert. Using another function to compute the expected value doesn't assure if the code we are testing is right and/or the test function is right.

Reply:
Yes, good point. I was trying not to change anything that was not either (1) directly related to the group-specific matching change, or (2) suggested by pylint. But I could try to improve the tests in general, if you'd like. Alternatively, I could make another PR.

Reply:
Having repeated code and assertion using strings is good. The test should be simple, it also clear that the invitation that you expect is the string that you are reading, it helps you to understand the behavior of the code. 
I suggest to go back to the original assertion:

```
assert client.get_invitation(id = 'auai.org/UAI/2019/Conference/Reviewers/-/Reviewing/Custom_Load')
assert client.get_invitation(id = 'auai.org/UAI/2019/Conference/Area_Chairs/-/Reviewing/Custom_Load')
```

Reply:
Ah, I see what you're saying now. I thought you didn't like the use of client.get_invitation(...).

I'll change this back